lab.js

```diff
@@ -0,0 +1,98 @@
+// Question 1
+
+const average = (num1, num2) => {
+    return (num1 + num2) / 2
+}
+console.log(average(2, 7))
+
+// Question 2 
+
+const average2 = (num3, num4, num5) => {
+    return (num3 + num4 + num5) / 3
+}
+console.log(average2(3, 3, 3))
+
+// Question 3
+
+const coolNum = (arr) => {
+    let sum = 0;
+    for (let i = 0; i < arr.length; i += 1) {
+        sum += arr[i]
+    }
+    return (sum / arr.length)
+}
+
+console.log(coolNum([1, 5, 2, 9, 4, 0]))
+
+//Question 4 
+
+const oddNums = (arr2) => {
+    let odds = []
+    for (let i = 0; i < arr2.length; i++){
+        if(arr2[i] % 2 !== 0){
+            odds.push(arr2[i])
+        }
+    }
+    return odds
+}
+console.log(oddNums([4, 5, 9, 10, 11]))
+
+
+// Question 5
+
+const posNum = (arr3) => {
+    for (let i = 0; i < arr3.length; i += 1) {
+        if (arr3[i] < 0) {
```

Comment:
The logic is all good, but you're returning a string that says 'true', or 'false', we want to return the boolean true, or false